opba-banking-rest-impl/src/main/java/de/adorsys/opba/tppbankingapi/controller/TppBankingApiAisController.java

```diff
@@ -47,6 +47,7 @@ public CompletableFuture getAccounts(
                                 .serviceSessionId(serviceSessionId)
                                 .requestId(xRequestID)
                                 .bankId(bankID)
+//                                .sessionPassword()
```

Comment:
Commented code? Probably you wanted to set password here

Reply:
removed